src/listeners/ready.ts

```diff
@@ -14,11 +15,30 @@ export class UserEvent extends Listener {
 		});
 	}
 
-	public run() {
+	public async run() {
+                this.container.logger.debug(await this.connectToSpreadsheet() == "200" ? "Connected to GoogleSheets" : "Connection to GoogleSheets Failed, you probably fucked something up")
```

Comment:
nice one

Reply:
looks pretty good (partly cuz I wrote half of it

Reply:
> 

🙆🏽‍♂️